docs/BuildOptions.md

```diff
@@ -0,0 +1,448 @@
+
+This option enables the logging instructions in the code that are considered
+heavy, they occur often and inform about very detailed parts of the library
+behavior (usually debug-level logs). Turning this option ON will allow you
+using the `debug` level of logging and get a very detailed information as to what
+happens inside the library. Note however that this may influence the processing
+by changing times, use less preferred thread switching layout, and generally
+worsen the functionality and performance of the library. Therefore this option
+is not turned on by default.
+
+
+`--enable-inet-pton` (default: ON)
+----------------------------------
+
+Enables usage of `inet_pton` function by the applications, which should be used
+to resolve the network endpoint name into an IP address. This may be not
+availabe on some version of Windows, in which case you can turn this OFF,
+however at the expense of not being able to resolve IP address by name,
+as the `inet_pton` function gets a poor-man's simple replacement that can
+only resolve numeric IPv4 addresses.
+
+
+`--enable-logging` (default: ON)
+-------------------------------
+
+Enables logging. When you turn this option OFF, logging will not be
+done at all. Could be tried when you suspect the logging system of
+impairing the performance, at the expense of not having some information
+about errors happening during runtime.
+
+
+`--enable-monotonic-clock` (default: OFF)
+-----------------------------------------
+
+Enforced `clock_gettime` with monotonic clock on Garbage Collector CV.
+**This is a temporary fix for #729**: The library could got stuck when you
+modify the system clock while a transmission is done over SRT.
+
+This option enforces the use of `clock_gettime` to get the current
+time, instead of `gettimeofday`. This function allows to force a monotonic
+clock, which is independent on the currently set time in the system. The CV,
+for which the `*_timedwait()` functions are used with so obtained current time,
+must be appropriately configured (and this is done so for now only for the
+GarbageCollector controlling CV, not every CV used in SRT). The consequence of
+enabling this option, however, may be portability issues around the
+`clock_gettime()` function, which is not available on every SDK, or extra
+`-lrt` option is sometimes required (this requirement will be autodetected).
+
+This option will be removed when the problem is fixed globally.
+
+
+`--enable-pktinfo` (default: OFF)
+---------------------------------
+
+This option allows extracting the target IP address from the incoming
+UDP packets and forceful setting of the source IP address in the outgoing
+UDP packets. This ensures that if a packet comes in from a peer that requests a
+new connection, the agent will respond with a UDP packet, which has the
+source IP address exactly the same as the one, which the peer is trying to
+connect to.
+
+When this is OFF, the source IP address in such outgoing UDP packet will
```

Comment:
@ethouris Shouldn't this be "ON"?

Reply:
No, this exactly describes what happens by default. But as you are mistaken here, users will be as well, so it likely needs to be said explicitly.